src/lib.js

```diff
@@ -26,6 +27,11 @@ const hypixel = axios.create({
 });
 const parseNbt = util.promisify(nbt.parse);
 
+let prices = await getPrices();
```

Comment:
I'm not 100% sure this is the best place for this... 

I have a feeling it should be similar to `src/scripts/update-bazaar.js`, opinions?

Reply:
First I thought about doing it like that but that would just slow down stuff since it has to take values from database. You choose, doesn't really matter

Reply:
I'll wait for the opinion of other @SkyCryptWebsite/developers